### ISSUE TITLE
Wrong TOC link to HTML content

### ISSUE THREAD

**Issue by koryukov** [EWSoftware/SHFB#611]
I would like to include documentation for my JavaScript library, that is generated by JSDoc, to main project documentation, that generated by SHFB. In accordance with solutions of issues #61 and #543 I have done the following:

* copy files from JSDoc output folder to *Content/JSDoc*:

```xml
<ItemGroup>
    <Folder Include="Content" />
    <Folder Include="Content\JSDoc" />
    <Folder Include="Content\JSDoc\fonts\" />
    <Folder Include="Content\JSDoc\scripts\" />
    <Folder Include="Content\JSDoc\scripts\prettify\" />
    <Folder Include="Content\JSDoc\styles\" />
    <Folder Include="Content\VersionHistory\" />
    <Folder Include="icons\" />
    <Folder Include="media\" />
</ItemGroup>
```
* apply **Content** build action to all files of *Content/JSDoc* folder:

```xml
<ItemGroup>
    <Content Include="Content\JSDoc\**\*" />
</ItemGroup>
```

* add following site map file to the documentation project:

```xml
<siteMap xmlns="http://schemas.microsoft.com/AspNet/SiteMap-File-1.0">
  <siteMapNode title="JavaScript API" url="Content\JSDoc\index.html" isSelected="true" />
</siteMap>
```

But it works only for CHM building. If I build documentation as an Website, the TOC link refers to */html/index.html* file, while the real file path is */Content/JSDoc/index.html*.

**Comment by EWSoftware** [comment]
Be aware that the sitemap referred to with regard to SHFB is not the same as a website sitemap and does not function the same way.  It's the equivalent of the content layout file for MAML topics.  It's just used to let the build engine know what files are included and how to lay them out in the TOC.  Add one to the project, open it in the editor, and define the layout of the topics similar to the way its done for MAML topics.

**Comment by EWSoftware** [comment]
Let me check into this.  I haven't used a sitemap with SHFB in years so I need to review how it's being used.

**Comment by koryukov** [comment]
Does the above description contain all information necessary to reproduce the problem? Or I need to 
provide additional details?

**Comment by EWSoftware** [comment]
I tried this and it works as expected.  This was resolved in v2017.12.30.0 which included a change to construct website TOC URLs in the VS2013 presentation style relative to the root instead of assuming the target is in the .\html folder (resolved issue #472).

Upgrade to the latest release and you should be able to generate the website content as you want it.

**Comment by koryukov** [comment]
Now the TOC title is overrided by *index.html* metadata, but I think, it's another issue. Thank you!
